src/doc/dev_manual/project_meetings.rst

```diff
@@ -0,0 +1,61 @@
+
+New Issue Triage
+----------------
+New issues are `triaged <https://www.bugsnag.com/blog/bug-triaging-best-practices>`_.
+New issues can be found through a browser using GitHub's `issue search feature <https://github.com/visit-dav/visit/issues?q=is%3Aissue+is%3Aopen+-label%3Areviewed>`_ and searching for issues missing the ``reviewed`` label. 
+During triage, issues are discussed in some detail to help refine meaning and determine ``impact`` (low, medium or high) and ``likelihood`` (low, medium or high) as well as other special considerations such as whether the issue is manifesting an outright ``crash``, should be given higher ``priority`` or may represent ``low hanging fruit`` (a small amount of work that will yield high dividends).
+Descriptions for the meaning of various labels to be applied to issues are provided on the `label description page <https://github.com/visit-dav/visit/labels>`_.
+Some issue submissions require lengthy discussion and dialog to help refine into one (or more) specific bug fixes or enhancement requests.
+Such issues are converted to discussions and should be gathered together to be discussed in a *special topics* meeting.
+
+Unanswered Q&A Discussion Triage
+--------------------------------
+Discussion items that have yet to be answered are triaged.
+Sometimes a new discussion will get converted to an issue.
+We don't currently use any labels for discussions but we have several discussion *categories* and we may re-categorize discussions to put them in the right category.
+We close any discussions which have been waiting for user response for more than 21 days.
+
+Special Topics Meeting Topics/Agenda
+------------------------------------
+We consider the need and interest in using the upcoming special topics meeting time.
+Sometimes, we opt to use this meeting time for further issue triage.
+Other times, we decide on some topic(s) to discuss and contributors may prepare some informal materials.
+
+Inclusive Moment
+----------------
+To begin the first project meeting of each new month a contributor shares an inclusive moment.
+An inclusive moment is a factoid, resource, event, experience, etc. related to STEAM with the purpose of raising awareness, seeding deeper thinking and/or informing future action.
+Example inclusive moments which are in scope include...
+
+* A short description of the book `"Black Software" <https://en.wikipedia.org/wiki/Black_Software>`_ and key take-aways to be learned from reading it.
+* A reminder and reference explaining how color choices in UI design impact color-blind users.
+* Relaying a recent conversation with a colleague that feels Fortran programmers aren’t sufficiently supported in ECP.
+  
+Example Inclusive Minutes which are out of scope...
+
+* Sharing a recent personal negative interaction with an airport security official
+
+  * **Rationale**: not related to STEAM.
+  
+* Playing the YouTube video `“Wny Diversity Matters?” <https://youtu.be/lHStHPQUzkE>`_.
```

Comment:
Should this be `youtube.com` instead of `youtu.be`? It appears to work, but it gets changed.

Reply:
What I have entered into the doc is from going to the video on YouTube and hitting the `Share` button and then hitting the `Copy link` button. Its a paste of whatever those steps produced and so I believe it is correct. I have seen other YouTube links with `youtu.be` in them too.